.github/workflows/test.yml

```diff
@@ -43,54 +43,50 @@ jobs:
       env:
         CI:
 
-  # Linux:
-  #   if: ${{ !(github.ref == 'refs/heads/master' || startsWith( github.ref, 'refs/tags/' )) }}
-  #   strategy:
-  #     fail-fast: false
-  #     matrix:
-  #       options:
-  #         - name: "Linux: gcc-11 ASAN No Toolchain"
-  #           run: ./housekeeping/make_build.sh -DCLEAR_OBJS=ON -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/gcc-11_cxx20.cmake -DASAN=ON
-  #         - name: "Linux: gcc-12 ASAN No Toolchain"
-  #           run: ./housekeeping/make_build.sh -DCLEAR_OBJS=ON -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/gcc-12_cxx20.cmake -DASAN=ON
-  #         - name: "Linux: clang-15 TSAN"
-  #           run: ./housekeeping/make_build.sh -DCLEAR_OBJS=ON -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/clang-15_cxx20.cmake -DTSAN=ON
-  #         - name: "Linux: clang-15 External Project"
-  #           run: ./housekeeping/make_external_build.sh -DCLEAR_OBJS=ON -DCMAKE_TOOLCHAIN_FILE=../../cmake/toolchain/clang-15_cxx20.cmake
-  #         - name: "Linux: clang-15 UBSAN"
-  #           run: ./housekeeping/make_build.sh -DCLEAR_OBJS=ON -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/clang-15_cxx20.cmake -DUBSAN=ON
+  Linux:
+    if: ${{ !(github.ref == 'refs/heads/master' || startsWith( github.ref, 'refs/tags/' )) }}
+    strategy:
+      fail-fast: false
+      matrix:
+        options:
+          - name: "Linux: gcc-12 ASAN No Toolchain"
+            run: ./housekeeping/make_build.sh -DCLEAR_OBJS=ON -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/gcc-12_cxx20.cmake -DASAN=ON
+          - name: "Linux: clang-15 TSAN"
+            run: ./housekeeping/make_build.sh -DCLEAR_OBJS=ON -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/clang-15_cxx20.cmake -DTSAN=ON
+          - name: "Linux: clang-15 External Project"
+            run: ./housekeeping/make_external_build.sh -DCLEAR_OBJS=ON -DCMAKE_TOOLCHAIN_FILE=../../cmake/toolchain/clang-15_cxx20.cmake
```

Comment:
Let's group sanitizers, and separate External project from them?

Reply:
Moved